specutils/tests/test_loaders.py

```diff
@@ -154,3 +160,234 @@ def test_no_reader_matches(name):
 
         with pytest.raises(IORegistryError):
             spec = Spectrum1D.read(filename)
+
+
+def test_iraf_linear():
+    test_file = CCDData(data=np.ones(4096),
```

Comment:
This should be composed outside of specutils, and then used as remote data.

Reply:
fixed as well